packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -778,6 +784,7 @@ class DateTimePicker extends React.Component {
 
         <div className={cx('date-facade')}>
           <DatePicker
+            a11yLabel={ariaLabel}
```

Comment:
There is an ariaLabel Prop in Date Picker new prop should not be needed.

Reply:
done https://github.com/cerner/terra-framework/pull/2031/commits/2d63b33d6e0de4b213da02c9b075de1cf5a96ae9